sdk/storage/Azure.Storage.Blobs/src/Generated/AppendBlobRestClient.cs

```diff
@@ -29,7 +29,7 @@ internal partial class AppendBlobRestClient
         /// <param name="clientDiagnostics"> The handler for diagnostic messaging in the client. </param>
         /// <param name="pipeline"> The HTTP pipeline for sending and receiving REST requests and responses. </param>
         /// <param name="url"> The URL of the service account, container, or blob that is the target of the desired operation. </param>
-        /// <param name="version"> Specifies the version of the operation to use for this request. The default value is "2021-12-02". </param>
+        /// <param name="version"> Specifies the version of the operation to use for this request. The default value is "2024-08-04". </param>
```

Comment:
@seanmcc-msft The rest api would return this header for version `2024-02-04` and above. This was the version we used when implementing the feature. Will that cause any issue in the SDKs?

Reply:
This is a generated file, and the comment is not accurate.